motoman_ms210_moveit_config/config/kinematics.yaml

```diff
@@ -0,0 +1,4 @@
+ms210_arm:
+  kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
```

Comment:
Please switch to Trac IK instead (and update the `run_depend`s of the package).

Reply:
Switched to Trac IK.
Note: Requires package `ros-<distro>-trac-ik-kinematics-plugin`